src/utils.py

```diff
@@ -0,0 +1,49 @@
+"""Utility classes and functions."""
+import re
+
+import daiquiri
+import requests
+
+from src.config import DAIQUIRI_LOG_LEVEL
+from src.gremlin import execute_query
+
+daiquiri.setup(level=DAIQUIRI_LOG_LEVEL)
+log = daiquiri.getLogger(__name__)
+
+
+def sanitize_text_for_query(text):
+    """Sanitize text so it can used in queries."""
+    if text is None:
+        return ''
+
+    if not isinstance(text, str):
```

Comment:
Since this is a hard requirement would be good to add this to the signature too:

`def sanitize_text_for_query(text: str) -> str:`

Reply:
done